mission_control/navigator_missions/nodes/run_mission

```diff
@@ -25,15 +25,15 @@ def main():
         print
         defer.returnValue(reactor.stop())
 
-    n = Navigator(nh)._init()
+    n = yield Navigator(nh)._init()
```

Comment:
If you put two missions in the command line, it would have tried to yield on the already yieled `navigator` object when the first mission completed.
Yieleding on a non `deffered` type object will throw errors